composer.json

```diff
@@ -22,7 +22,8 @@
         "illuminate/support": "~5.3.0 || ~5.4.0 || ~5.5.0",
         "illuminate/pagination": "~5.3.0 || ~5.4.0 || ~5.5.0",
         "nesbot/carbon": "~1.0",
-        "graphaware/neo4j-bolt": "dev-feature/causal-cluster-support"
+        "graphaware/neo4j-bolt": "dev-feature/causal-cluster-support",
+        "fzaninotto/faker": "~1.4"
```

Comment:
Should be included in dev dependencies only
@iyhunko

Reply:
@IlliaBalia thanks